### ISSUE TITLE
Communicating the Institutional Partner Programme

### ISSUE THREAD

**Issue by drjwbaker** [programminghistorian/jekyll#1658]
@jenniferisasi You mentioned in the call doing more to spread the word regarding how people can support us. Now we're going ahead with https://github.com/programminghistorian/jekyll/issues/1625 can we have a think about what to do to get this out there.

A few thoughts:

- direct mail to a list of institutions/libraries/research groups we think might want to contribute (perhaps make a list on a google doc?)
- a slide for PH members when giving talks that summarises ways people/groups can support us.
- some regular tweeting with library/open access related hashtags

**Comment by drjwbaker** [comment]
@jenniferisasi IPP is live https://github.com/programminghistorian/jekyll/pull/1656 Any initial thoughts on a comms plan?

**Comment by jenniferisasi** [comment]
Sorry, I had lost sight of this issue for some reason. But yes, I agree with your three ideas above. In my mind, the firsts institutions we should contact are the ones we work in (only when confortable doing it, of course) and the popular ones that we can think of. 

We can also check Analytics to see areas where a lot of users are connecting from and check the libraries there? (Sounds a bit surveillance but worth a look?)

**Comment by drjwbaker** [comment]
Thanks @jenniferisasi. Agreed.

I've asked my library colleagues for good UK/European lists to spread the word about the IPP. @svmelton: any good north american library lists you could recommend?

We could also look at those institutions that support comparable initiatives.

On analytics, good idea. I'll mention to Penny.

Finally, we'll need to agree some text to send around (I guess a condensed version of what is on the website).

**Comment by jenniferisasi** [comment]
Oh, and I can certainly email my postdoc granting council ([CLIR](https://www.clir.org/)) as they partner with the Digital Library Federation, etc. and ask if they could support us with $ but also for some clues as to whom to ask or to distribute it with their postdoc-alumni, once we have a text.

**Comment by drjwbaker** [comment]
Note @jenniferisasi that at https://github.com/programminghistorian/jekyll/issues/1594 @rivaquiroga agreed to help you think through the IPP comms.

**Comment by acrymble** [comment]
Just a reminder that the calendar available on #1554 shows how we planned to promote Patreon. it's worked fairly well. Something similar for IPP might be useful, to make sure the right audiences are targeted but not repeatedly so.

**Comment by svmelton** [comment]
@drjwbaker I'm not sure about specific lists, but I agree that checking with CLIR/DLF would be a good start. I can think of some potential libraries to target if we decide to go down that route.

**Comment by acrymble** [comment]
I'd like to suggest this come with some reasonable goals for number of partners in the first year, which we can work towards together.

**Comment by drjwbaker** [comment]
Can we link it to items on the business plan @acrymble ? 2 IPPs makes us financially sustainable based on our current ongoings, so that is not much of a target. So if there are specific items on the business plan (or coming out of https://github.com/programminghistorian/jekyll/issues/1489) we want to link our goals to, that would make sense. The one the comes to mind for me is how many IPPs do we need to hire a Project Manager (as a heads-up, Penny is scoping that because that - *spoilers* - is their headline finding).

**Comment by acrymble** [comment]
Once you have something ready @rivaquiroga @jenniferisasi can you let me know so I can post it to Patreon? I think the people there would be likely interested in promoting this option with their libraries.

**Comment by acrymble** [comment]
Some additional resources / ideas:

OLH (Open LIbrary of Humanities) is offering a referral discount: https://www.openlibhums.org/plugins/supporters/referrals/codes/ which presumably encourages librarians to do some of the promotion work for you.

The OLH institutional signup page is here: https://www.openlibhums.org/plugins/supporters/signup/


@jenniferisasi can you help us to define what you're expecting to produce to resolve this ticket? I want to make sure we're on the same page. I'm hoping it will be something persuasive that can easily be disseminated to potential partners along with a strategy for reaching them.

@drjwbaker can we also be clear if we're waiting on this strategy before reaching out to potential partners? The earlier in the year we can start reaching out, the less difficult it will be to secure repeat signups. Keep in mind that in the UK at least the budget year end is July.

**Comment by jenniferisasi** [comment]
@acrymble I was thinking to follow through the three main ideas @drjwbaker had already suggested in his first post: an email, tweets, encourage PH editors to have a slide for conferences. 

I can start a list on a Google doc of institutions we can contact. Now, there is no point in doing the contact in the next few weeks until people settle in their day to day work at home or go back to offices. This is something external to their everyday duties and might be just disregarded.

**Comment by acrymble** [comment]
Thanks @jenniferisasi for clarifying. You're right about matters of timing. I don't work in a library but I think it's quite central to *someone's* job in most academic libraries. We just need to learn who that is.

**Comment by drjwbaker** [comment]
On https://github.com/programminghistorian/jekyll/issues/1691#issuecomment-598381364 @acrymble can we wait on contacting Jisc until @jenniferisasi has developed generic wording for inviting people to the IPP?

**Comment by rivaquiroga** [comment]
I agree with @jenniferisasi about the timing. It might be disregarded, as many libraries are currently concentrated in supporting the transition to online learning in their institutions. But in a couple of weeks or a month they might be interested, as our resources are valuable for online learning.
I would like to suggest developing a prospectus for possible partners the way conferences do to find sponsors. Here is an example: https://user2018.r-project.org/img/useR2018sponsorship.pdf. That way the email is short, and all the details are in the document. I can work on the design if someone can help with the wording.

**Comment by drjwbaker** [comment]
@rivaquiroga That is a fabulous idea.

**Comment by jenniferisasi** [comment]
I had a chance to look at the example you provided @rivaquiroga and I really like it. We already have the [specifics of the IPP](https://programminghistorian.org/en/support-us#institutional-partner-programme) on our site, and in my mind I was thinking an email + those details + a link to the site. Do we want to create a PDF too? 

(Working on the email; sent you a link to document)

**Comment by acrymble** [comment]
I know we're heading into a long weekend for many of us, but I wonder if we can set a realistic deadline for when we hope to have this messaging ready to distribute? Then we can talk about who will contact whom and such. @drjwbaker is end of April 2020 reasonable? I believe you've mostly got the messaging done, so it's a matter of design with @rivaquiroga . Is that right?

**Comment by drjwbaker** [comment]
@jenniferisasi is leading on this, so it is down to her capacity right now.

**Comment by acrymble** [comment]
My mistake. @jenniferisasi is the end of April reasonable for you and @rivaquiroga ?

And @drjwbaker just so everyone is clear. Is @jenniferisasi also going to be reaching out to potential IPP organisations, or does that fall back to your remit? Either way, do we need to open a separate ticket for the strategy on that? It seems distinct from the text itself.

**Comment by drjwbaker** [comment]
@jenniferisasi is Comms Lead, so unless she requests for help from us, I expect that having drafted the mailout, Jen will write to identified orgs from our shared gmail account, and I will deal with responses.

**Comment by jenniferisasi** [comment]
Sorry I'm just catching up on this. The message is almost ready (I left a couple of comments) and I believe the end of the month is reasonable (gonna block some time on the calendar). What I need to know, because I am never sure... who in the library would be getting the emails? @drjwbaker you prob can help me, what title did the people who contact you have?

**Comment by drjwbaker** [comment]
@jenniferisasi Absolutely no hurry. We all need not to put too much pressure on each other at this time. My instinct is library directors. In some cases, I've suggested specific people where I know they are likely to be supportive.

**Comment by acrymble** [comment]
@jenniferisasi I think @svmelton and @walshbr might be the best people to ask about that. At least in the American context.

**Comment by drjwbaker** [comment]
> @jenniferisasi I think @svmelton and @walshbr might be the best people to ask about that. At least in the American context.

Yes, especially on specific people at specific places.

**Comment by walshbr** [comment]
Hmm…sort of depends on the angle I suppose. Digital scholarship librarians, digital humanities librarians, digital humanities coordinators, and people in those types of positions would be the most engaged in questions of open access. But if the library has subject liaisons, then the history department liaison or someone who educates on digital literacy might be a good bet. My sense is that each library can vary a lot in terms of what kinds of positions exist or not, but those might be a decent starting point. But yeah as @drjwbaker says I suppose you could always start at the top with library directors. An intermediate step would be Associate University Librarians, who might have the sorts of people I list above as reporting to them.

**Comment by svmelton** [comment]
Library directors would be the simplest, and the most likely to be able to make those decisions. @walshbr is right, though—the people most engaged will likely be DH librarians, scholarly communications librarians, and liaison librarians. One solution might be to send the letter to directors, but also email listservs where those on-the-ground library staff are likely to be.

**Comment by drjwbaker** [comment]
Thanks @walshbr @svmelton. For the institutions on the gdoc, if you know specific people who'd be good/sympathetic "ins" to an org, please add them. Otherwise, let's go with Library Directors.

**Comment by jenniferisasi** [comment]
As indicated to @acrymble on #1756 I am working on collecting the emails to send it out as soon as I can. Upon reading the google doc again, I want to clarify: 

Our journals can be added to the libraries catalogs without being a IPP member, so I added a sentence that at least could help make us more findable.

**Comment by drjwbaker** [comment]
@jenniferisasi Thanks for your efforts on this. I know you've had **a lot** going on. I think it is moving along nicely.

**Comment by drjwbaker** [comment]
And I just gave the email a re-read. Looking great.

**Comment by acrymble** [comment]
@drjwbaker can you please help me understand why you've removed the project management label from this? I have renamed it from "very old" to "3 months+" to recognize that my phraseology was perhaps poorly thought out. But from the perspective of someone whose role is to identify areas of the project in need of additional support, this feels passive aggressive (which I'm sure you don't intend it to), but it makes me not want to continue with my role (and this has happened elsewhere).

**Comment by drjwbaker** [comment]
"3 months+" is much better. Thanks. "Very old" just made it seem like we weren't working on it, which we are.

**Comment by drjwbaker** [comment]
I should add. No harm meant. Sincere apology if it was caused.

**Comment by drjwbaker** [comment]
@jenniferisasi I think we can close this as you've done the first comms round. Please reopen if you feel it should be reopened!
